drinks_touch/notifications/notification.py

```diff
@@ -66,8 +66,9 @@ def send_notification(to_address, subject, content_text, content_html, uid):
 
 
 def send_drink(user, drink, with_summary=False):
+    account = Account.query.filter(Account.email == user["email"]).one()
```

Comment:
How about selecting by ldap_id?

```suggestion
    account = Account.query.filter(Account.ldap_id == user["id"]).one()
```